dev-php/composer/composer-1.2.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils versionator
+
+DESCRIPTION="A dependancy manager for PHP"
+HOMEPAGE="http://getcomposer.org"
+
+SRC_URI="http://getcomposer.org/download/${PV}/${PN}.phar"
```

Comment:
This leaves composer.phar without version in the Manifest. To avoid problems with bumps, it would be best to rename this to something like ${P}.phar.